testcases.py

```diff
@@ -151,18 +151,64 @@ def _keylog_file(self) -> str:
             return self._server_keylog_file
         logging.debug("No key log file found.")
 
+    def _inject_keylog_if_possible(
+        self, trace_raw: str, trace_with_secrets: str
+    ) -> str:
+        """
+        Inject the keylog file into a pcap file if it is available and valid.
+
+        :Return: ``trace_with_secrets`` if it was available or ``trace_raw`` if not.
+        """
+        keylog = self._keylog_file()
+
+        if (
+            os.path.isfile(trace_with_secrets)
+            and os.path.getsize(trace_with_secrets) > 0
+        ):
+            # already injected
+
```

Comment:
Why would this happen?

Reply:
It's a long time since I wrote this. I think it was an edge case in my specific setup where I experimented with trace files. I guess you can remove it.
Feel free to take over the work on this PR. I work on different topics and can't spend time on it anymore, sorry.